deblur/parallel_deblur.py

```diff
@@ -48,7 +48,8 @@ def deblur_system_call(params, input_fp):
                '--keep-tmp-files']
     command.extend(params)
 
-    logger.debug('running command %s' % command)
+    logger.info('[%s] running command %s' % (mp.current_process().name,
```

Comment:
I think we should keep this as debug rather than info (info should show the main step restuls - details are usually in the debug level?)